data/coalition/resistance intro.txt

```diff
@@ -1718,19 +1873,20 @@ mission "Resistance: Join"
 	to offer
 		has "Resistance: Quarg Interview: done"
 	on offer
+		log "Recorded Oobat interviewing a Quarg. It went well, until she asked it if thy wished to reclaim the rings. It said they did, because they are their homes. It seemed to be pained by this thought."
```

Comment:
```suggestion
		log "Recorded Oobat interviewing a Quarg. It went well, until she asked it if they wished to reclaim the rings. It said they did, because they are their homes. It seemed to be pained by this thought."
```